tests/test_manual_connect.c

```diff
@@ -1,19 +1,53 @@
-/*
- * Copyright (c) 2018-2024 Snowflake Computing, Inc. All rights reserved.
- */
+//
```

Comment:
why do we need to change the comments format here?

Reply:
I reverted back.